### PR TITLE
feat: table

### PR DESCRIPTION
**Pull request by cuzuco2**
Implementation fo table using spectrum css

## How Has This Been Tested?

Implementation 
https://devsite-665-table--adobe-io-website--adobe.hlx.page/photoshop/index-new

## Screenshots (if appropriate):

image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: hlx_statics/blocks/table/table.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    if (i) tbody.append(row);
+    else thead.append(row);
+    [...child.children].forEach((col) => {
+      const cell = buildCell(i);
+      cell.classList.add('spectrum-Table-cell');
+      cell.innerHTML = col.innerHTML;
+      row.append(cell);
+    });
+  });
+  block.innerHTML = '';
```
Comment (AndreiAlexandruParaschiv) [inline hlx_statics/blocks/table/table.js:33]: Better to use `block.textContent = '';` as .innerHTML presents a potential security risk (XSS attack)

**Comment by solaris007** [comment]
the table looks different from the spectrum template. also, i think that we don't need any specific CSS, as spectrum should provide all borders and colors:
- markup looks different (e.g. td instead of th used in thead)
- border colors too dark
- column heading fonts look different
- center table on page and add margin

Screenshot 2023-02-22 at 07 50 15
Screenshot 2023-02-22 at 07 53 07

suggestion: remove all custom CSS and use spectrum classes only

**Comment by solaris007** [comment]
also in mobile sizes, the table doesn't go into scrolling mode with nice margins/padding

**Review by solaris007 — APPROVED** [review]
great job

**Review by AndreiAlexandruParaschiv — APPROVED** [review]

**Pull request merged**